src/lib/get-osm-stream.js

```diff
@@ -1,22 +1,24 @@
 'use strict'
 const url = require('url')
-const request = require('request')
-const Readable = require('stream').Readable
+const http = require('http')
 const { osmapi } = require('../config')
 
 // Query the osm api using a bounding box string, returns a readable stream
 module.exports = function getOsmStream (bbox, cb) {
```

Comment:
Looks good! If you wanted to avoid needing a callback, you could use a `through` stream and do something like this:

```js
const http = require('http')
const through = require('through2')

module.exports = function getOsmStream (bbox) {
  const queryUrl = url.resolve(osmapi, `map?bbox=${bbox}`)
  const t = through()

  http.get(queryUrl, res => {
    res.pipe(t)

    if (res.statusCode !== 200) {
      t.emit('error', new Error('bad status code ' + res.statusCode))
    }   
  })  

  return t
}
```